checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInference.java

```diff
@@ -323,9 +395,13 @@ private void inferOwningField(Node node, MethodInvocationNode invocation) {
   }
 
   /**
-   * Analyzes an assignment statement and performs three computations:
+   * Analyzes an assignment statement and performs four computations:
```

Comment:
Rather than changing this from 3 to 4, maybe change this to be non-specific about the size of the list, so that we don't have to update it in the future: `performs the following computations:`